.github/workflows/icu_merge_ci.yml

```diff
@@ -36,3 +36,163 @@ jobs:
+            flag: "-l -u"
+          - perf: normperf
+            tests: TestICU_NFC_NFD_Text TestICU_NFC_NFC_Text TestICU_NFC_Orig_Text TestICU_NFD_NFD_Text TestICU_NFD_NFC_Text TestICU_NFD_Orig_Text
+        exclude:
+          - perf: collperf
+            data: th18057
+          - perf: collperf
+            data: thesis
+          - perf: collperf
+            data: vfear11a
+          - perf: ustrperf
+            data: TestNames_Japanese
+          - perf: ustrperf
+            data: thesis
+          - perf: ustrperf
+            data: vfear11a
+          - perf: normperf
+            data: TestNames_Simplified_Chinese
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Create directory for lib files
+        run: mkdir icu4c/source/perflib
+      - name: Get ICU libs
+        uses: actions/download-artifact@v2
+        with:
+          name: icu-perf-libs
+          path: icu4c/source/lib
+
+      - name: Build and run ICU4C perf tests
+        env:
+          DATA_FILE_PATH: ../../icu4j/perf-tests/data/collation
+        run: |
+          cd icu4c/source;
+          ./runConfigureICU Linux;
+          (cd test/perf && make -j2 everything && mkdir -p results && mkdir -p results/${{ matrix.perf }} && mkdir -p results/${{ matrix.perf }}/${{ matrix.data }})
+          LD_LIBRARY_PATH=lib ./test/perf/${{ matrix.perf }}/${{ matrix.perf }} ${{ matrix.flag }} -t 5 -p 10 -f $DATA_FILE_PATH/${{ matrix.data }}.txt ${{ matrix.tests }} | tee test/perf/results/${{ matrix.perf }}/${{ matrix.data }}/output.txt
+
+      - name: Store performance test results
+        uses: gregtatum/github-action-benchmark@d3f06f738e9612988d575db23fae5ca0008d3d12
+        with:
+          # The perf tests result data is in ndjson format.
+          tool: 'ndjson'
+          output-file-path: icu4c/source/test/perf/results/${{ matrix.perf }}/${{ matrix.data }}/output.txt
+          # Tentative setting.
+          alert-threshold: '100%'
+          fail-on-alert: true
+          gh-pages-branch: perfdata
+          benchmark-data-dir-path: perf/results/${{ matrix.perf }}/${{ matrix.data }}
+          auto-push: true
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          comment-on-alert: true
+
+ # Retrieve performance test results and upload results to remote repository
+  perf-test-data-move:
+    name: Copy perf data to remote repo for visualization
+    needs: [icu4c-performance-tests, icu4c-performance-tests-with-files]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Switch to benchmark data storage branch.
+        run: |
+          git fetch
+          git checkout perfdata
+
+      - name: Publish data
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          deploy_key: ${{ secrets.ACTIONS_DEPLOY_KEY }}
+          external_repository: gnrunge/icu-perf
```

Comment:
Now you need to replace the external repo coordinates with the proper `unicode-org/___` repo identifier.

Reply:
Switched to branch main in repository unicode-org/icu-perf.